### PR TITLE
[DOCU-2357]Overview Dashboard page for Konnect

### PR DESCRIPTION
**Pull request by Guaris**
### Summary
Request from Christian. 

### Reason
[Jira](https://konghq.atlassian.net/browse/DOCU-2357?atlOrigin=eyJpIjoiNTRjMTcyYTkyYzcxNGZkZWExYTA3OWE3M2I0NGU4MWMiLCJwIjoiamlyYS1zbGFjay1pbnQifQ)

### Testing
[Overview Dashboard docs
](https://deploy-preview-4013--kongdocs.netlify.app/konnect/vitals/overview-dashboard/)

### CODE REVIEW

File: app/konnect/vitals/analytics-overview.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: Overview Dashboard
+---
+
+The Overview Dashboard provides metrics for Services catalogued by Service Hub within a selected time interval for the following two categories: 
+
+* **Traffic** - total number of HTTP requests.
+* **Errors** - average error rate in percentage across every HTTP requests.
```
Comment (sichvoge) [inline app/konnect/vitals/analytics-overview.md:8]: Is it "request" or "requests"?

**Comment by sichvoge** [comment]
@Guaris Only two comments. Otherwise this looks good. I'll ask @ryanmoore to also take a look.

File: app/konnect/vitals/analyze.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Analyze Services and Routes
 no_version: true
+alpha: true
```
Comment (mheap) [inline app/konnect/vitals/analyze.md:4]: @Guaris What does this entry in the front matter mean?

File: app/konnect/vitals/analytics-overview.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+Last 6 hours | Data is aggregated in ten minute increments.
+Last 12 hours| Data is aggregated in ten minute increments. 
+Last 24 hours| Data is aggregated in ten minute increments. 
+Last 7 days | Data is aggregated in one hour increments. 
+Last 30 days | Data is aggregated in one hour increments.
+
+{:.important}
+> Free tier users can only select intervals up to 24 hours.
+
+Selecting a different time period changes all of the metrics within the selected category to match the selected period, **except for trends**.
```
Comment (mheap) [inline app/konnect/vitals/analytics-overview.md:38]: It feels like we're missing something here to explain why trends isn't impacted

> except for trends, which XYZ

Comment (Guaris) [reply app/konnect/vitals/analytics-overview.md]: It's for tech debt reasons.

File: app/konnect/vitals/overview-dashboard.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+---
+title: Overview Dashboard
+no_version: true
+content_type: reference
+---
+
+The Overview Dashboard provides metrics for Services catalogued by Service Hub within a selected time interval for the following two categories: 
+
+* **Traffic** - total number of HTTP requests.
+* **Errors** - percentage of failed HTTP requests.
```
Comment (lena-larionova) [inline app/konnect/vitals/overview-dashboard.md:10]:
```suggestion
* **Traffic**: Total number of HTTP requests.
* **Errors**: Percentage of failed HTTP requests. 
```
I'd go with colons for consistency's sake. That's the style we use most frequently in the kong docs.

**Comment by Guaris** [comment]
Note to self: 

Remove line 38. 

Trends are now time period/over time period so 15 minutes/previous 15 minutes 30 days over/30 days and not week over week exclusively.

**Review by lena-larionova — APPROVED** [review]
LGTM other than the line that needs to be removed.

File: app/konnect/vitals/analytics-overview.md (hunk 3 above)
Comment (Guaris) [reply app/konnect/vitals/analytics-overview.md]: we've solved this.

**Pull request merged**